src/assessment.js

```diff
@@ -27,7 +28,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = ['deathStar', 'fire', 'global'];
```

Comment:
No, `global` scope cannot access `laserQuantity`. This is because `laserQuantity` is enclosed within the `deathStar` function.